AUTHORS.txt

```diff
@@ -49,3 +49,4 @@ Jack Van Sambeek
 JC Arbelbide
 Florian Jünger
 Benedikt Moneke
+Fabio Garzetti
```

Comment:
Please update also docs/about/authors.rst.
Unfortunately we have this duplication.

Reply:
Done